DiffusionPreprocessing/DiffPreprocPipeline_Eddy.sh

```diff
@@ -379,7 +379,9 @@ get_options() {
 	echo "   resamp_value: ${resamp_value}"
 	echo "   ol_nstd_value: ${ol_nstd_value}"
 	echo "   extra_eddy_args: ${extra_eddy_args}"
-	echo "   no_gpu: ${no_gpu}"
+	if [ ! -z ${no_gpu} ]; then
```

Comment:
I think our convention in other scripts is to print argument values even if they are empty or used the default.

Reply:
Ok, I've adjusted this to print all argument values. To make the printed output more meaningful I set the default value from no_gpu (and SelectBestB0) to false rather than an empty string.